public/angular/js/controllers/media/settings.js

```diff
@@ -0,0 +1,52 @@
+angular.module('Aggie')
+
+.controller('MediaSettingsController', [
+  '$scope',
+  'Settings',
+  '$timeout',
+  '$filter',
+  'FlashService',
+  'mediaSettingsOptions',
+  'MediaSettingsModal',
+  function($scope, Settings, $timeout, $filter, flash, mediaSettingsOptions, modal) {
+
+    $scope.media = {};
+    var item;
+    var mediaName;
+    for (item in mediaSettingsOptions) {
```

Comment:
`mediaSettingsOptions` is an array, right? I don't think you're supposed to use `for`...`in` with an array. http://stackoverflow.com/questions/500504/why-is-using-for-in-with-array-iteration-such-a-bad-idea